README.md

```diff
@@ -1,39 +1,36 @@
 [![Build Status](https://travis-ci.org/stone-payments/kong-plugin-url-rewrite.svg?branch=master)](https://travis-ci.org/stone-payments/kong-plugin-url-rewrite)
 
-# kong-plugin-url-rewrite
+# Kong-plugin-url-rewrite
 
-This is a Kong middleware to log requests / responses using cerberus API.
+Kong API Gateway plugin for url-rewrite purposes.
 
-# project structure
+## The Problem
 
-The plugin folder should contain at least a `schema.lua` and a `handler.lua`, alongside with a `spec` folder and a `.rockspec` file specifying the current version of the package.
-
-# rockspec format
-
-The `.rockspec` file should follow [LuaRocks' conventions](https://github.com/luarocks/luarocks/wiki/Rockspec-format)
+When using Kong, you can create routes that proxy to an upstream. The problem lies when the upstream has an url that is not very friendly to your clients, or restful, or event pretty. When you [add a Route in Kong](https://docs.konghq.com/0.14.x/admin-api/#add-route), you have a [somewhat limited](https://docs.konghq.com/0.14.x/proxy/#routes-and-matching-capabilities) url rewrite capability. This plugin simply throws away the url set in Kong route and uses the url set in it's configuration to proxy to the upstream. This gives you full freedom as to how to write your url's in Kong and inner services as well.
```

Comment:
you meant - **even** pretty instead of event?

Reply:
Yes. :frowning_woman: